lib/handlers/delete.js

```diff
@@ -16,9 +16,7 @@ function handler(req, res) {
     ldp.delete(filename, function(err) {
         if (err) {
             debug("DELETE -- error: " + err);
-            return res
-                .status(err.status)
-                .send(err.message);
+            return next(err);
```

Comment:
I am sure. I tried your example first but the requests are not being intercepted. The request is immediately sent. You need to call next(err) for the error handler to be used.